### DISCUSSION TITLE
ZepetoPlayers 의 중력을 없앨 수 있나요?

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#348] [5. Q&A]
제목 그대로 ZepetoPlayers 의 중력을 없앨 수 있나요?

**Comment by moondory77-naverz** [comment]
안녕하세요.
(float) ZepetoPlayers.gravity 값을 변경함으로써,
해당 월드에서, 캐릭터에 적용되는 중력 값을 제어 가능합니다.

해당 (중력)값은 ZepetoCharacter 컴포넌트에서 제어하는 Move 동작에 적용되는 값으로, 사용에 유의해주세요.
감사합니다.
